basic tutorials (#1)
keypom/keypom-docs: docs/Tutorials/Basics/getting-started.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem'; 

# Getting Started

> In the basic tutorials, you will be shwon how to create [the four types](/Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/introduction.md) of Keypom drops. Two methods will be shown.

Author: added explanation of the drop between the first sentence and the "throughout each tutorial, two simple NodeJS scripts"

1) [NEAR-API-JS](https://docs.near.org/tools/near-api-js/reference)  
2) [Keypom-JS SDK](https://github.com/keypom/keypom-js) <-- currently linked to github, change this to typedocs or smth else once ready)

In the Deploy scripts, the same four types of Keypom drops will be created, but you will be able to customize the configurations to your needs. 

---

## Prerequisites
For the basic tutorials, you can choose to run the scripts on your own machine. To do son, you must have the following:

1. [Node JS](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)  
2. [NEAR-API-JS](https://docs.near.org/tools/near-api-js/quick-reference#install)  

> 3. To install the SDK, simply run the following in your command prompt.  

Author: docs.keypom.xyz? do we have/need a "how to install" page?

> 3. To install the SDK, simply run the following in your command prompt.  

Reviewer: Just link out to the README - oop just checked and we don't have an install guide. I'll add that to the README now and then report back on this comment with the link.

> 3. To install the SDK, simply run the following in your command prompt.  

Reviewer: @minqianlu https://github.com/keypom/keypom-js#installation

```bash
npm -i keypom-js
```

At this point, you are ready to start the tutorials. To interact with the deploy scripts, you can continue reading this page.  

For the Deploy Scripts, you should have a [NEAR account](https://docs.near.org/concepts/basics/account). It can be either `testnet` or `mainnet`.
:::note
If you choose to use mainnet, you will be using real $NEAR to fund your drops.
:::

---

> ## Deploy Scripts

Author: we can put any deploy script stuff in its own category if you want. That way we create the separation b/n the tutorials and the deploy scripts.

> ## Deploy Scripts

Reviewer: Yes because docs shouldn't really be referencing the deploy scripts, those belong to the protocol and so we should add this documentation that you have into the readme.

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem'; 

# Getting Started

> The primary goal of a Keypom drop is to send assets to new *or* existing users. These assets can take on many forms.

Reviewer: ```suggestion
The primary goal of a Keypom drop is to send assets to users with the click of a link regardless of whether they have a wallet or not. These assets can come in many forms which will be broken down below.
```

> The basic tutorials are meant as introductory guides that will help you create your first Keypom drops and understand the differences between the [types](/Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/introduction.md) of Keypom drops and the assets they send. 

Reviewer: ```suggestion
The basic tutorials are meant as introductory guides that will help you create your first Keypom drops and understand the differences between the [types](/Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/introduction.md) of Keypom drops and the assets they send. 
```
```suggestion
The basic tutorials are meant as introductory guides that will help you create your first Keypom drops and understand the differences between the [types](/Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/introduction.md) of Keypom drops and the assets they contain. 
```

> There are **four** types of Keypom drops. Each drop type holds different types of assets and can be sent to new **or** existing users.

Reviewer: ```suggestion
There are **four** types of drops, each with different types of assets. These drops can be claimed either with a brand new NEAR account or an existing one.
```

> | Drop Type                 | Assets Embedded in Web2 Style Link                                                                                     |

Reviewer: ```suggestion
| Drop Type                 | Assets                                                                                   |
```

| ------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
| [Simple Drop](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/simpledrops.md)             | $NEAR                              |

> | [Non-Fungible Token Drop](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/nftdrops.md)    | $NEAR and an NFT                   |

Reviewer: ```suggestion
| [Non-Fungible Token Drop](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/nftdrops.md)    | NFT and optionally $NEAR                |
```

> | [Fungible Token Drop](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/ftdrops.md)         | $NEAR and FTs                      |

Reviewer: ```suggestion
| [Fungible Token Drop](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/ftdrops.md)         | Fungible Tokens and optionally $NEAR                     |
```

> | [Function Call Drop](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/fcdrops.md)          | $NEAR and a set of callable methods|

Reviewer: ```suggestion
| [Function Call Drop](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/fcdrops.md)          | Set of function-calls and optionally $NEAR|
```

> The [Simple Drop](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/simpledrops.md) is the most basic type of drop, it allows you to $NEAR to new or existing users. 

Reviewer: ```suggestion
The [Simple Drop](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/simpledrops.md) is the most basic type of drop, it allows you to send $NEAR via a link to new or existing users. 
```

> [NFT Drop](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/nftdrops.md) and [FT Drop](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/ftdrops.md) are extensions of the Simple drop, they allow you to send NFTs and FTs in addition to $NEAR. 

Reviewer: ```suggestion
[NFT Drops](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/nftdrops.md) and [FT Drops](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/ftdrops.md) are extensions of the Simple drop, they allow you to send NFTs and FTs in addition to $NEAR. 
```

> The [Function Call Drop](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/fcdrops.md) is Keypom's most powerful type of drop. They allow you send a link that is capable of calling almost any function on any smart contract that you predefine. A simple example of this is lazy-minting NFTs, where the NFT is minted once the user claims their key. 

Reviewer: ```suggestion
The [Function Call Drop](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/fcdrops.md) is Keypom's most powerful type of drop. It allows you send a link that is capable of calling almost any function on any smart contract that you predefine. A simple example of this is auto-registering users into a DAO as part of the onboarding process.
```
